container/src/LuigiContainer.svelte

```diff
@@ -151,6 +152,15 @@
       webcomponentService.thisComponent = thisComponent;
 
       const ctx = GenericHelperFunctions.resolveContext(context);
+
+      thisComponent.updateViewUrl = (viewUrl: string, internal?: object) => {
+        if (viewUrl?.length) {
+          const context = Object.assign(ctx, { viewUrl });
+
+          ContainerAPI.updateViewUrl(context, internal, iframeHandle);
```

Comment:
```suggestion
          ContainerAPI.updateViewUrl(viewUrl, context, internal, iframeHandle);
```
since this is a newly introduced internal function, imho no need to "hide" the most relevant argument in the generic context object ;)